src/idm/group.js

```diff
@@ -0,0 +1,125 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+
+const group = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @summary List all groups
+    * @name getGroups
+    * @public
+    * @function
+    * @memberof agile.idm.group
+    * @fulfil {Array} all groups
+    * @returns {Promise}
+    * @example
+    * agile.idm.group.getGroups().then(function(groups) {
```

Comment:
Yes but that doesn't make it right, the sdk is a chance to make the interaction between all agile services uniform. Naming is hard but in this case it looks obvious

`agile.idm.group.get()` looks far better than `agile.idm.group.get()`

Also this isn't a direct mapping of the API, for instance looks at [the device manager](https://github.com/Agile-IoT/agile-sdk/blob/master/DOCUMENTATION.md#agile.deviceManager.get) there we get all devices or if an id is specified return the single instance. It decreases the sdk's api surface area and makes things cleaner.